### ISSUE TITLE
[FEATURE] A better navigation icon 

### ISSUE THREAD

**Issue by ChinmayMhatre** [EddieHubCommunity/LinkFree#1672]
### Description

I looked up the website while adding my own profile. The magnifying glass icon didn't seem very intuitive to me at first. Couldn't figure out at first glance that that's where I needed to go to look at all the profiles. My suggestion would be a button telling that's where all the profiles are.

### Screenshots

![image](https://user-images.githubusercontent.com/51131670/186836202-2c03024d-00ca-4478-b79e-e6679709b32d.png)

**Comment by Njong392** [comment]
I kind of agree. I initially thought that was a search bar, and only after clicking in the icon itself did I really understand what it was for. Maybe we could switch up the approach to letting people know that that's how you can view all the profiles.

**Comment by nathgoutam93** [comment]
Imo, the landing page should have a call to action button instead of "see eddie jaoude's profile as an example",
saying something like "see profiles 🔎"  or  "search for profiles 🔎" then "Want to add your profile? Read the instructions" below.

**Comment by schmelto** [comment]
> Imo, the landing page should have a call to action button instead of "see eddie jaoude's profile as an example", saying something like "see profiles 🔎" or "search for profiles 🔎" then "Want to add your profile? Read the instructions" below following that.

I like the idea of a call to action.

**Comment by AkshithBellare** [comment]
I think because the entire navbar looks like a search bar, first-time visitors might take some time to realise only the icon is clickable. Adding the phrase "search for profiles" might make it more obvious.

**Comment by Njong392** [comment]
> I think because the entire navbar looks like a search bar, first-time visitors might take some time to realise only the icon is clickable. Adding the phrase "search for profiles" might make it more obvious.

I think the search bar thing should be removed altogether, maybe go for a button or better still, a CTA like @nathgoutam93 suggested because it's not serving as a search bar even though it looks like one.

**Comment by Vedant-Manjrekar** [comment]
@schmelto  @ChinmayMhatre i would like to work on this issue.

**Comment by Vedant-Manjrekar** [comment]
@schmelto how is this, 
1. Added **search profiles** next to the search icon for better understanding of its use case as mentioned by @AkshithBellare
2. A **"see profiles 🔎 for inspiration"** instead of "See profile for an example" which redirects user to the search profile page as mentioned by @nathgoutam93 

Ready add a PR if changes are approved.

<img width="1065" alt="Screenshot 2022-10-01 at 1 08 41 PM" src="https://user-images.githubusercontent.com/93585735/193398673-50c4be0a-076e-44d9-9c96-162be3482088.png">

**Comment by krshkodes** [comment]
Looks good 👍

**Comment by Vedant-Manjrekar** [comment]
May i make a PR ?

**Comment by krshkodes** [comment]
Sure @Vedant-Manjrekar, I that for any issue you first see if anyone else isn't working and ask for assignment on the issue and open the PR!

**Comment by eddiejaoude** [comment]
The new version of LinkFree is out, I will close this issue, feel free to create a new issue for the new version if still relevant
